x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -78,6 +80,8 @@ export class AlertsClient<
   private fetchedAlerts: {
     indices: Record<string, string>;
     data: Record<string, Alert & AlertData>;
+    seqNo: Record<string, number | undefined>;
```

Comment:
Here is where we capture the seqNo/primaryTerm for the previous alert docs, in case we need to update them.